level1/main.tf

```diff
@@ -1,10 +1,26 @@
+# module "vpc" {
```

Comment:
could you please remove commented code from file

Reply:
done